internal/pkg/core/monitoring/config/config.go

```diff
@@ -33,6 +36,7 @@ type MonitoringConfig struct {
 	MonitorTraces    bool                  `yaml:"traces" config:"traces"`
 	APM              APMConfig             `yaml:"apm,omitempty" config:"apm,omitempty" json:"apm,omitempty"`
 	Diagnostics      Diagnostics           `yaml:"diagnostics,omitempty" json:"diagnostics,omitempty"`
+	RuntimeManager   string                `yaml:"_runtime_experimental,omitempty" config:"_runtime_experimental,omitempty"`
```

Comment:
Since this is more of an enumeration that supports only a finite set of values (currently two), I'd prefer if we used a custom type here instead of `string`.  Then we could add validation on this type when a string is being unmarshalled to it, and tests around the unmarshalling.

Reply:
Could we perhaps use the `component.RuntimeManager` type here?

Reply:
I originally intended to, but https://github.com/elastic/go-ucfg/issues/169 makes doing to impossible.